### PR TITLE
Create glossary.md

### PR DESCRIPTION
**Pull request by rjchow**
Two reasons to write this document:

1) Standardising our terminology so that we know we're referring to the same things
2) Helping newcomers get up to speed with all the terminogy

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  
+  Envelope: Wrapper around a document that uses OpenAttestation framework to protect its integrity. _(Sometimes called Signed JSON file)_
+  
+  Proof: Part of the `Envelope` metadata, a set of Keccak256 hashes that leads to the Merkle root that is notarised (i.e on the Ethereum blockchain)
+  
+  Document: The information inside the .opencert or .tt file that excludes the `Envelope` metadata. _(Sometimes called Unsigned JSON)_
+  
+  Schema: Refers to a specific schema that the contained `Document` has to adhere to.
+  
+  Template: Instructions on how to display the data in the `Document`, could be a URL pointing to a `Renderer` or simply a path depending on OpenAttestation version
```
Comment (yehjxraymond) [inline glossary.md]:
```suggestion
  Template: Instructions on how to display the data in the `Document`, it is currently an URL pointing to a `Renderer`. For OpenCerts, a simple path is used for v1.4 & v1.5, prior to that, a key is used.  
```

Comment (yehjxraymond) [reply glossary.md]: We have two legacy in OpenCerts. 
Also wanted to show that going with the path/key is **not the norm**

File: glossary.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  
+  Renderer: A HTTP(s) endpoint that takes a `Document` and returns some browser-renderable contents (i.e HTML/CSS/JS)
```
Comment (yehjxraymond) [inline glossary.md]:
```suggestion
  Renderer: A website to be embedded in the OpenAttestation platform. It should take a `Document` and render it in browser. A sample application is available in the `decentralized-renderer` repository.
```

File: glossary.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  
+  RendererFrame: A component on the OpenCerts/OpenAttestation site that takes a `Document` and sends it to the `Renderer` endpoint.
```
Comment (yehjxraymond) [inline glossary.md]: This is confusing, people don't need to know about this also.

File: glossary.md (hunk 2 above)
Comment (yehjxraymond) [reply glossary.md]: Wanted to highlight that is is actually rendered.

File: glossary.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+### OpenCerts
+
+  SSG: SkillsFuture Singapore, the owner of OpenCerts
```
Comment (yehjxraymond) [inline glossary.md]:
```suggestion
  SSG: SkillsFuture Singapore, the owner of OpenCerts accreditation registry
```

File: glossary.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  
+  SSG Registry: A list of entities considered to be trustworth issuers, administered by SSG _(Sometimes called The Registry or just Registry)_
```
Comment (yehjxraymond) [inline glossary.md]:
```suggestion
  SSG Registry: A list of accredited institutes, administered by SSG _(Sometimes called The Registry or just Registry)_
```

File: glossary.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  
+  Issuer: An entity that issues educational credentials, represented by one or more DocumentStores.
+  
+  Certificate: .OpenCert file that contains a verifiable educational credential.
+  
+  DocumentStore: An Ethereum smart contract that keeps track of OpenCerts issuances and revocations.
+  
+  DocumentStore Address: The specific Ethereum Address that a `DocumentStore` is located at.
```
Comment (yehjxraymond) [inline glossary.md]: Wanna differentiate with the wallet address ?

File: glossary.md (hunk 2 above)
Comment (rjchow) [reply glossary.md]: technically the browser renders it, the url just takes the document and returns some code for the browser to render

File: glossary.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+OpenAttestation: The underlying technology/framework that we use to power verifiable credentials
+
+OpenCerts: The project that is owned by SSG and is concerned with educational credentials
+
+TradeTrust: The project that is owned by IMDA and is concerned with trade documents as well as transferable asset deeds
+
+### OpenAttestation Framework
+  
+  Envelope: Wrapper around a document that uses OpenAttestation framework to protect its integrity. _(Sometimes called Signed JSON file)_
```
Comment (Nebulis) [inline glossary.md]: WrappedDocument ?

**Pull request closed without merging**